runtime/vm/BytecodeInterpreter.cpp

```diff
@@ -36,7 +36,11 @@
 
 /* USE_COMPUTED_GOTO on Windows is a performance improvement of 15% */
 /* USE_COMPUTED_GOTO on Linux amd64 has a performance improvement of 3% */
-#if (defined(WIN32) && defined(__GNUC__)) || (defined(LINUX) && defined(J9HAMMER))
+#if (defined(WIN32) && defined(__GNUC__))
+#define USE_COMPUTED_GOTO
+#elif (defined(LINUX) && defined(J9HAMMER))
+#define USE_COMPUTED_GOTO
+#elif defined(OSX)
```

Comment:
Why not just tack this on the end of the existing line?

Reply:
I felt like the line was getting long and difficult to read. I can switch it back if you prefer